radio/src/targets/horus/CMakeLists.txt

```diff
@@ -80,6 +112,16 @@ if (PCB STREQUAL X10)
     option(INTERNAL_MODULE_MULTI "Support for MULTI internal module" OFF)
     option(BLUETOOTH "Support for bluetooth module" OFF)
     add_definitions(-DMANUFACTURER_JUMPER)
+# //OW
+    if (TELEMETRY_MAVLINK)
+      option(INTERNAL_GPS "Support for internal ublox GPS" OFF)
```

Comment:
I don't see why this is specific to MAVLink. You could want to have internal GPS support even with no MAVLink.